packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -2488,6 +2488,56 @@ void main() {
     expect(widths.indexOf(largestWidth), equals(1));
   }, skip: isBrowser); // https://github.com/flutter/flutter/issues/39998
 
+  testWidgets('showTimeSeperator is only supported in time or dateAndTime mode', (
+    WidgetTester tester,
```

Comment:
I think this could just be `test` instead of `testWidgets`, since it does not use `tester`.